README.md

```diff
@@ -129,11 +129,12 @@ sources:
     schema: <schema_name>
     database: <database_name>
     loader: fivetran
-    loaded_at_field: _fivetran_synced
-      
-    freshness: # feel free to adjust to your liking
-      warn_after: {count: 72, period: hour}
-      error_after: {count: 168, period: hour}
+    
+    config:
```

Comment:
Looks like there's an `enabled` argument in the `src_mixpanel.yml` ([link)](https://github.com/fivetran/dbt_mixpanel/blob/main/models/staging/src_mixpanel.yml#L12), should we include that in this README section?

Reply:
Good question .... Since this is for the user-created sources, we don't need that `enabled` flag in this case. It serves to turn off the default mixpanel source for union-folks -- otherwise, they have a floating source in their DAG. So shouldn't be applicable to their own sources!